### PR TITLE
Add tooltips to column headers

### PR DESCRIPTION
**Pull request by dalepotter**
# #72 TO BE MERGED BEFORE THIS ONE

Adds tooltips to column headers.

Note some of the tooltips do not appear in line with the cell header text, for example:

image

This can likely be resolved by upgrading to Bootstrap v4 and using the `offset` option: https://getbootstrap.com/docs/4.0/components/tooltips/#options however this I would suggest this is not MVP for this feature.

~Temporary live preview: http://61830dae.ngrok.io/dashboard~ < No longer available.

### CODE REVIEW

**Review by wendyrogers — APPROVED** [review]

File: src/templates/dashboard.html [hunk 1]
```diff
@@ -27,16 +28,10 @@ <h2></h2>
...
-					<th class="sort" data-sort="comprehensive">Comprehensive</th>
-					<th class="sort" data-sort="coverage">Coverage</th>
-					<th class="sort" data-sort="humanitarian">Humanitarian</th>
-					<th class="sort" data-sort="total">Total</th>
-					<th class="sort" data-sort="progress">Progress</th>
-					<th class="sort" data-sort="baseline">Baseline</th>
+					<th class="sort" data-sort="organisation" data-toggle="tooltip" data-placement="bottom" title="Name of the organisation or agency that is a signatory or a signatory affiliate of the Grand Bargain">Organisation</th>
+					<th class="sort" data-sort="first-published" data-toggle="tooltip" data-placement="bottom" title="The date when the organisation first published to IATI">First Published</th>
+					<th class="sort" data-sort="progress" data-toggle="tooltip" data-placement="bottom" title="‘Total’ value at 1 June 2017">Progress</th>
+					<th class="sort" data-sort="baseline" data-toggle="tooltip" data-placement="bottom" title="Total (current) value minus Baseline value">Baseline</th>
```
Comment (hayfield) [inline src/templates/dashboard.html]: One of the `Total`s in tooltips is quoted, the other isn't.

File: src/templates/dashboard.html [hunk 2]
```diff
@@ -52,14 +47,48 @@ <h2></h2>
...
+	<div role="tabpanel" class="tab-pane" id="details-tab">
+		<h2></h2>
+		<p>This page provides individual assessment values for the five performance measures (identified within the framework and methodology for measuring the quality of humanitarian data published to IATI). Please see the 'Methodology' tab for details on how each measure is assessed or hover over column headings for a brief description.</p>
+
+		<table class="table">
+			<thead>
+				<tr>
+					<th class="sort" data-sort="organisation" data-toggle="tooltip" data-placement="bottom" title="Name of the organisation or agency that is a signatory or a signatory affiliate of the Grand Bargain">Organisation</th>
+					<th class="sort" data-sort="timeliness" data-toggle="tooltip" data-placement="bottom" title="How often does the organisation update its data (% value)?">Timeliness</th>
+					<th class="sort" data-sort="forward-looking" data-toggle="tooltip" data-placement="bottom" title="How many of the organisation’s current projects or interventions include annual planned financial flows ie activity budgets (% value)?">Forward Looking</th>
```
Comment (hayfield) [inline src/templates/dashboard.html]: Mac-like tick in `organisation's`

File: src/templates/dashboard.html [hunk 3]
```diff
@@ -52,14 +47,48 @@ <h2></h2>
...
+					<th class="sort" data-sort="comprehensive" data-toggle="tooltip" data-placement="bottom" title="How many of the IATI fields that most data users want to see are being published by the organisation for its current activities (% value)?">Comprehensive</th>
+					<th class="sort" data-sort="coverage" data-toggle="tooltip" data-placement="bottom" title="How much of its total operational portfolio is the organisation publishing information to IATI about (% value)?">Coverage</th>
+					<th class="sort" data-sort="humanitarian" data-toggle="tooltip" data-placement="bottom" title="How many of the organisation’s published current activities contain specific humanitarian-related information eg which emergency(s), appeals, clusters etc.">Humanitarian</th>
+					<th class="sort" data-sort="total" data-toggle="tooltip" data-placement="bottom" title="Aggregated assessment value average which combines the  percentage values for all five measures (1. Timeliness, 2. Forward Looking, 3. Comprehensive, 4. Coverage, 5. Humanitarian - see ‘Details’ tab)">Total</th>
```
Comment (hayfield) [inline src/templates/dashboard.html]: Strange ticks around `Details`

**Review by hayfield — CHANGES_REQUESTED** [review]

File: src/templates/dashboard.html (hunk 1 above)
Comment (dalepotter) [reply src/templates/dashboard.html]: Fixed in c01a6de

File: src/templates/dashboard.html (hunk 2 above)
Comment (dalepotter) [reply src/templates/dashboard.html]: Fixed in c01a6de

File: src/templates/dashboard.html (hunk 3 above)
Comment (dalepotter) [reply src/templates/dashboard.html]: Fixed in c01a6de

**Review by hayfield — APPROVED** [review]
:ticket:

**Pull request merged**